src/java.base/share/classes/java/lang/classfile/components/CodeRelabeler.java

```diff
@@ -64,7 +68,7 @@ static CodeRelabeler of(Map<Label, Label> map) {
     /**
      * Creates a new instance of CodeRelabeler using provided {@link java.util.function.BiFunction}
      * to re-label the code.
-     * @param mapFunction
+     * @param mapFunction function remapping labels
```

Comment:
I found this a bit hard to read. Maybe expand to "function for remapping labels in the source code model", or something similar

Reply:
Fixed, thanks.